src/components/slideshow/slideshow.js

```diff
@@ -481,6 +545,9 @@ export default function (options) {
      * Closes the dialog and destroys the Swiper instance.
      */
     function onDialogClosed() {
+        // Exits fullscreen
+        fullscreenExit();
```

Comment:
In TV layout, this gives an error here https://github.com/jellyfin/jellyfin-web/blob/f7f96bee73127c5e0699ea03e549fa9374426c87/src/components/slideshow/slideshow.js#L510
because buttons were not created.

Reply:
Thanks again, forgot the `if(btnFullscreen)` before it, did the same for the exit fullscreen button. :)